src/main/java/org/mamute/list/TabsHelper.java

```diff
@@ -0,0 +1,47 @@
+package org.mamute.list;
+
+import br.com.caelum.vraptor.http.route.Router;
+import net.vidageek.mirror.dsl.Mirror;
+import org.apache.commons.lang.WordUtils;
+import org.mamute.controllers.ListController;
+
+import javax.inject.Inject;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+public class TabsHelper {
```

Comment:
In that case I could create a ConnectionProvider class or so, which is specific enough for others to know what it is supposed to do. My point is just that I don't like helper classes :) 
Maybe if we find a better name for this one, we can keep it that way